source/openpulse/openpulse/__init__.py

```diff
@@ -0,0 +1,26 @@
+"""
+===================================
+OpenPulse Python reference package
+===================================
+
+This package contains the reference abstract syntax tree (AST) for representing
+OpenPulse 3 programs, tools to parse text into this AST, and tools to manipulate
```

Comment:
Do we refer to it as "OpenPulse 3" or just "OpenPulse"?

Reply:
Good catch! Fixed.